qc2md.py

```diff
@@ -229,6 +245,7 @@ def get_dialogue_lines_at_time(
 def write_markdown(
     output_filename: str | None,
     entries: dict[str, list[QCEntry]],
+    nondialogue_categories: list[str],
```

Comment:
Same as with `standalone_categories` in `categorize_entries`, unsure about making this a mandatory argument, as it has no effect unless `include_references` is enabled.